hack/go.sh

```diff
@@ -86,8 +90,9 @@ linters(){
 ci-checks(){
     echo "-------- beginning preflight checks"
     lint
-    test
-    build
+    test || RET=$?
```

Comment:
perhaps safer to define `local RET=0` in all those functions instead of setting a global var?

Reply:
Yeah, It is preferable. I'll fix them